discover/deployment/articles/02-configuring-liferay/02-document-repository.markdown

```diff
@@ -222,10 +222,11 @@ You must include this property in `system-ext.properties`:
     org.xml.sax.driver=com.sun.org.apache.xerces.internal.parsers.SAXParser
 
 Other app servers also need this configuration if they don't contain a
-`SAXParser`. 
+`SAXParser`. Remember to place the `system-ext.properties` in your @product@
+installation's `/WEB-INF/classes` directory.
 
```

Comment:
I see. I'm not aware of any other way unfortunately. Either way, system-ext doesn't seem to be picked up when in Liferay Home, which is the main point of my pull request.